opdevnet/opdevnet_test.go

```diff
@@ -135,6 +135,60 @@ func TestOPDevnet(t *testing.T) {
 		}
 		time.Sleep(l2SlotDuration)
 	}
+
+	// Set up and run a verifier node
+	verifierL2EngineURL, err := e2eurl.ParseString("ws://127.0.0.1:8894")
+	require.NoError(t, err)
+	verifierOpNodeURL, err := e2eurl.ParseString("http://127.0.0.1:8895")
+	require.NoError(t, err)
+	verifierL2EthURL, err := e2eurl.ParseString("ws://127.0.0.1:8896")
+	require.NoError(t, err)
+
+	verifierL2Node, verifierL2Backend, err := geth.InitL2("l2-verifier", l2Genesis.Config.ChainID, l2Genesis, writeJWT(t, jwtSecret), func(_ *ethconfig.Config, nodeCfg *node.Config) error {
+		nodeCfg.AuthAddr = verifierL2EngineURL.Hostname()
+		nodeCfg.AuthPort = int(verifierL2EngineURL.PortU16())
+		nodeCfg.WSHost = verifierL2EthURL.Hostname()
+		nodeCfg.WSPort = int(verifierL2EthURL.PortU16())
+		return nil
+	})
+	require.NoError(t, err)
+	require.NoError(t, verifierL2Node.Start())
+	defer func() {
+		require.NoError(t, verifierL2Node.Close())
+	}()
+	l2EngineURL.IsReachable(ctx)
+
+	verifierOpConfig, err := opdevnet.BuildOPConfig(
```

Comment:
Any thoughts on putting this in a separate `TestVerifier` test function? It will run in parallel in that case, and might be a bit easier to read.

Reply:
I like the idea of separating them out into subtests. I'll look into splitting these up and running them in parallel

Reply:
I looked into this for a little bit and had a setup that used a channel to store the sequencer state root at block 10 from the existing test for comparison in the new verifier test. Unfortunately there was still a lot of setup logic that had to be copied over to the `TestVerifier` test and I feel like it made it less readable. I'm going to opt to leave this in one test unless you feel strongly about splitting it up